api/src/main/java/com/epam/pipeline/dao/pipeline/PipelineRunDao.java

```diff
@@ -93,6 +96,9 @@ public class PipelineRunDao extends NamedParameterJdbcDaoSupport {
     @Autowired
     private RunServiceUrlDao serviceUrlDao;
 
+    @Autowired
```

Comment:
would it be possible to inherit from this class instead of have it as dep? And then, you can move `1686-1689` lines to `JdbcTemplateReadOnlyWrapper`. it should slightly reduce number of copy-paste lines in a future, when we will reuse this approach for other DAOs, if needed.

P.S. with this approach it would be better to rename `JdbcTemplateReadOnlyWrapper` as `JdbcTemplateWithDryRunWrapper`